tests/test_emsi_client.py

```diff
@@ -245,3 +246,34 @@ def test_get_job_postings_data_error(self):
                 )
         ):
             self.client.get_job_postings(RankingFacet.TITLE_NAME, JOB_POSTINGS_FILTER)
+
+    @mock_api_response(
+        method=responses.POST,
+        url=EMSIJobsApiClient.API_BASE_URL + '/taxonomies/{}/lookup'.format(RankingFacet.TITLE.value),
+        json=JOB_LOOKUP,
+    )
+    def test_get_details(self):
+        """
+        Validate that the behavior of client while fetching job lookup data.
+        """
+        jobs_details = self.client.get_details(RankingFacet.TITLE, JOB_LOOKUP_FILTER)
+
+        assert jobs_details == JOB_LOOKUP
+
+    @mock_api_response(
+        method=responses.POST,
+        url=EMSIJobsApiClient.API_BASE_URL + '/taxonomies/{}/lookup'.format(RankingFacet.TITLE.value),
+        json=JOB_LOOKUP,
+        status=400,
+    )
+    def test_get_details_error(self):
+        """
+        Validate that the behavior of client when error occurs while fetching job lookup data.
+        """
+        with raises(
+                TaxonomyAPIError,
```

Comment:
Indentation seems to be a bit out of place here

Reply:
This is expected Indentation here. Otherwise, we get a quality error.

Reply:
(bad-continuation)